setup.py

```diff
@@ -17,8 +17,16 @@
     entry_points={
         "lexibank.dataset": [
             "kpaamcam=lexibank_kpaamcam:Dataset"
-        ]
+        ],
+        'cldfbench.commands': [
+            'lowerfungom=kpaamcamcommands',
+        ],
     },
-    install_requires=["pylexibank>=2.6.0"],
+    install_requires=[
+        "pylexibank>=2.6.0", 
+        "python-igraph", 
+        "matplotlib",
+        "scipy",
+        "pandas"],
```

Comment:
I've added all non-default requirements for the `lingpy` functionality used here.